src/controller/WebController.php

```diff
@@ -429,7 +414,8 @@ public function invokeGlobalSearch($request)
     public function invokeVocabularySearch($request)
     {
         $template = $this->twig->loadTemplate('vocab-search.twig');
-        $this->setLanguageProperties($request->getLang());
+        //$this->setLanguageProperties($request->getLang());
```

Comment:
same as above, remove

Reply:
Removed in new commit: https://github.com/NatLibFi/Skosmos/commit/48b31a51acb2fd8be2452c033e0cecd05d86c764